docs: comparision between community and subscription repositories (#1333)
bareos/bareos: docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

   systemctl restart apache2 php-fpm

> Console/Profile changes
> ~~~~~~~~~~~~~~~~~~~~~~~
> 
> The |webui| Director profile :config:option:`dir/profile = webui-admin` has been extended by the directive :config:option:`dir/profile/PluginOptionsACL = *all*` in Bareos :sinceVersion:`22.0.0: PluginOptionsACL = *all* added to profile webui-admin`.
> 
> The reason for this is because you are now able to provide a plugin options string in the restore form of |webui| in order to restore jobs that used a plugin during backup.
> If you want to make use of that feature in |webui|, a proper configured :config:option:`dir/profile/PluginOptionsACL` is required.

Reviewer: ```suggestion
Console/Profile changes
~~~~~~~~~~~~~~~~~~~~~~~

The |webui| restore form now accepts a `plugin options string` in the restore form for plugin restores. 
Therefore, the |webui| Director profile :config:option:`dir/profile = webui-admin` has been extended by the directive :config:option:`dir/profile/PluginOptionsACL = *all*` in Bareos :sinceVersion:`22.0.0: PluginOptionsACL = *all* added to profile webui-admin`.
To make use of that feature in |webui|, a proper configured :config:option:`dir/profile/PluginOptionsACL` is required.
```
